docs/core/toc.yml

```diff
@@ -7,477 +7,483 @@ items:
+      href: testing/unit-testing-code-coverage.md
+    - name: Unit test published output
+      href: testing/unit-testing-published-output.md
+    - name: Live unit test .NET Core projects with Visual Studio
+      href: /visualstudio/test/live-unit-testing-start
 - name: Versioning
   items:
-  - name: Overview
-    href: versions/index.md
-  - name: .NET Core version selection
-    href: versions/selection.md
+    - name: Overview
+      href: versions/index.md
+    - name: .NET Core version selection
+      href: versions/selection.md
 - name: Runtime Identifier (RID) catalog
   href: rid-catalog.md
 - name: Resource manifest names
   href: resources/manifest-file-names.md
 - name: Dependency management and loading
   items:
-  - name: Dependency management
-    href: tools/dependencies.md
-  - name: Dependency loading
-    href: dependency-loading/overview.md
-  - name: Understand AssemblyLoadContext
-    href: dependency-loading/understanding-assemblyloadcontext.md
-  - name: Dependency loading details
-    items:
-    - name: Default dependency probing
-      href: dependency-loading/default-probing.md
-    - name: Load managed assemblies
-      href: dependency-loading/loading-managed.md
-    - name: Load satellite assemblies
-      href: dependency-loading/loading-resources.md
-    - name: Load unmanaged libraries
-      href: dependency-loading/loading-unmanaged.md
-  - name: Tutorials
-    items:
-    - name: Create a .NET Core application with plugins
-      href: tutorials/creating-app-with-plugin-support.md
-    - name: How to use and debug assembly unloadability in .NET Core
-      href: ../standard/assembly/unloadability.md
+    - name: Dependency management
+      href: tools/dependencies.md
+    - name: Dependency loading
+      href: dependency-loading/overview.md
+    - name: Understand AssemblyLoadContext
+      href: dependency-loading/understanding-assemblyloadcontext.md
+    - name: Dependency loading details
+      items:
+        - name: Default dependency probing
+          href: dependency-loading/default-probing.md
+        - name: Load managed assemblies
+          href: dependency-loading/loading-managed.md
+        - name: Load satellite assemblies
+          href: dependency-loading/loading-resources.md
+        - name: Load unmanaged libraries
+          href: dependency-loading/loading-unmanaged.md
+    - name: Tutorials
+      items:
+        - name: Create a .NET Core application with plugins
+          href: tutorials/creating-app-with-plugin-support.md
+        - name: How to use and debug assembly unloadability in .NET Core
+          href: ../standard/assembly/unloadability.md
 - name: .NET Core distribution packaging
   href: distribution-packaging.md
 - name: Expose .NET Core components to COM
   href: native-interop/expose-components-to-com.md
 - name: Host .NET Core from native code
   href: tutorials/netcore-hosting.md
+- name: Packages, metapackages, and frameworks
+  href: packages.md
```

Comment:
:warning: This has a build warning (invalid link)

Reply:
That looks like a merge conflict, the article was retired, these lines should be removed